modules/enabling-multi-cluster-console.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * assemblies/web-console.adoc
+
+:_content-type: PROCEDURE
+[id="enable-multi-cluster-console_{context}"]
+= Enabling multicluster in the web console
+
+:FeatureName: Multicluster console
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+.Prerequisites
+* Your cluster must be using the latest version of {product-title}.
+* You must have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/install/index[Red Hat Advanced Cluster Management (ACM) for Kubernetes 2.5] installed.
+* You must have administrator privileges.
+
+WARNING: Do not set this feature gate on production clusters. You will not be able to upgrade your cluster after applying the feature gate, and it cannot be undone.
+
+.Procedure
+
+. Log in to the {product-title} web console using your credentials.
+
+. Enable ACM in the administrator perspective by navigating from *Admin* -> *ClusterSettings* -> *Configuration* -> *Console Details* -> *Console Plugins* and click `enable`.
+
+. Click *Refresh the web console* in the pop-up window that states that a web console update is available.
+** *local-cluster* and *All Clusters* is visible above the perspectives in the navigation section.
+
+. Enable the feature gate navigating from *Administration* -> *Cluster Settings* -> *Configuration* -> *FeatureGate*, and edit the YAML template as follows:
++
+[source,yaml]
+
+----
+spec:
+    featureSet: TechPreviewNoUpgrade
+----
+
+. Click save to enable the multicluster console for all clusters.
++
+IMPORTANT: After you save, this feature is enabled and cannot be undone.
```

Comment:
Same with this admonition https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#admonitions